app.go

```diff
@@ -320,6 +320,20 @@ type Config struct { //nolint:govet // Aligning the struct fields is not necessa
 	// Default: json.Unmarshal
 	JSONDecoder utils.JSONUnmarshal `json:"-"`
 
+	// When set by an external client of Fiber it will use the provided implementation of a
```

Comment:
@imsk17 pls add this to the docs
https://github.com/gofiber/fiber/blob/main/docs/api/fiber.md#config-fields

Reply:
solved with [`fc1779b` (#3173)](https://github.com/gofiber/fiber/pull/3173/commits/fc1779b6df4602e0acd42e5695fe9cda7d439880)